src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -166,7 +166,7 @@ inline bool is_power_activation(const ov::Node* node) noexcept {
             return true;
         }
         return (1.0f != value);
-    } else if (auto power_op = std::dynamic_pointer_cast<ngraph::op::PowerIE>(node)) {
+    } else if (auto power_op = ov::Any(node).as<std::shared_ptr<ngraph::op::PowerIE>>()) {
```

Comment:
looks strange that we need to wrap to ov::Any first.
Why do we need to change something here?

Reply:
> looks strange that we need to wrap to ov::Any first. Why do we need to change something here?
**std::dynamic_pointer_cast**<ngraph::op::PowerIE>(node)) used here is not the standards func from std, which is is the api (deprecated) defined in Any. Maybe we can convert it by dynamic_case like this 
`auto power_op = dynamic_cast<const ngraph::op::PowerIE*>(node)`
How do u think?